src/components/metadataFormControls/CategoryComboSelect/useOptionsQuery.ts

```diff
@@ -62,7 +62,11 @@ export function useOptionsQuery() {
                     const { id, displayName, isDefault } = catCombo
                     return {
                         value: id,
-                        label: isDefault ? i18n.t('None') : displayName,
+                        // This should be distinguishable from other selects
+                        // where "none" means no selection
+                        label: isDefault
+                            ? i18n.t('Default (none)')
+                            : displayName,
```

Comment:
I think we should check if this is what we want, since it was just `None` in the old app. We should probably not render both though (havent tested if it does).

Reply:
I copied the values from the old app. Could change them in the future, but I suppose the copying the old labels should suffice for now and won't be confusing to anyone who has used the old app already.